js/components/terms-and-conditions/terms-and-conditions.js

```diff
@@ -45,7 +43,7 @@ define([
     checkAcceptance() {
       const acceptanceDate = localStorage.getItem('terms-and-conditions-acceptance-date');
       if (acceptanceDate !== null) {
-        const isExpired = momentApi.diffInDays(parseInt(acceptanceDate, 10), momentjs().add(appConfig.termsAndConditions.acceptanceExpiresInDays, 'days')) <= 0;
+        const isExpired = momentApi.diffInDays(parseInt(acceptanceDate), Date.now()) > appConfig.termsAndConditions.acceptanceExpiresInDays;
```

Comment:
But why you removed the radix parameter?
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/parseInt

Reply:
fixed